build-parent/pom.xml

```diff
@@ -1132,6 +1132,45 @@ To suppress enforce-helm-version rule, you must add following plugin to the root
                 <hide.manual.actions>true</hide.manual.actions>
             </properties>
         </profile>
+
+        <!-- Docker host configuration by detecting common docker socket locations -->
+        <profile>
+            <!-- Rancher Desktop on Linux/Mac -->
```

Comment:
I: It would be trivially easy to include Docker Desktop as well, but since we strongly recommend Rancher Desktop I didn't add it.